<a href="https://colab.research.google.com/github/LuisDelgado09/Matematicas-Aplicadas/blob/main/_1_Practica_de_escritura.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

**INTEGRANTES**

- Luis Miguel Delgado Moreno A00410889
- Geremy Javier Añazco Hernandez A00411261
- Nicolas Erazo Velez A00411299

Punto 1.1

$$r = \frac{A}{2\pi g}$$

Punto 1.2

$$h = \frac{3V}{A_b}$$

Punto 1.3

$$r = \sqrt[3]{\frac{3V}{4\pi}}$$

Punto 1.4

$$h = \frac{2A}{B + b}$$

Punto 1.5

$$t = \frac{q}{I}$$

Punto 1.6

$$R = \frac{V}{I}$$

Punto 1.7

$$t = \frac{E R}{V^2}$$

Punto 1.8

$$R = \frac{1}{\frac{1}{R_1} + \frac{1}{R_2}}$$

Punto 1.9

$$P_f = \frac{P_0 V_0 T_f}{T_0 V_f}$$

Punto 1.10

$$V_B = V_A - I R$$










Punto 2

In [ ]:
y= float ((3 * (-5 + 10 + 5 * (-5)) - 2 * ((-2) + (-4) + 10)) ** 2)
print (y)

4624.0


In [ ]:
f= (45 + (3* (2 + 8) -8/4) + 10 - 83)
print (f)

0.0


In [ ]:
a= (((-4+7)*2-10)*(-4)-7-3)
print (a)

6


Punto 3

$$\frac{1}{5}[(\frac{1}{2})^2+(\frac{3}{4})^2]+\frac{10}{25}[(\frac{1}{2})^2+(\frac{3}{4})^2]+\frac{8}{40}[(\frac{1}{2})^2+(\frac{3}{4})^2]$$

In [ ]:
PuntoTres = (1/5) * ((1/2)**2 + (3/4)**2) + (10/25) * ((1/2)**2 + (3/4)**2) + (8/40) * ((1/2)**2 + (3/4)**2)
print(PuntoTres)

0.65


$$\frac{9}{4}(\frac{6}{8})^{-1}-\frac{5}{2}(\frac{25}{20})^{-1}+\frac{40}{8}(\frac{20}{16})^{-1}$$

In [ ]:
PuntoTres2= (9/4) * (6/8)**-1 - (5/2) * (25/20)**-1 + (40/8) * (20/16)**-1
print(PuntoTres2)

5.0


Punto 4.

In [ ]:
nombre = input("Cual es tu nombre? ")
peso = int(input("Cual es tu peso? "))
talla = float(input("Cual es tu talla? en metros "))
imc = peso / (talla ** 2)
print(f"Tu imc es {imc}")

Cual es tu nombre? nicolas
Cual es tu peso? 64
Cual es tu talla? en metros 1.71
Tu imc es 21.887076365377382


In [ ]:

a = 1/2
b = 3/4


expr1 = (1/5) * (a**2 + b**2) + (10/25) * (a**2 + b**2) + (8/40) * (a**2 + b**2)


expr2 = (9/4) * (6/8)**-1 - (5/2) * (25/20)**-1 + (40/8) * (20/16)**-1


print("Resultado de la primera expresión:", expr1)
print("Resultado de la segunda expresión:", expr2)


Resultado de la primera expresión: 0.65
Resultado de la segunda expresión: 5.0


PUNTO 5

In [ ]:
Numero1= int(input("Ingrese el primer número:"))

Numero2= int(input("Ingrese el segundo número:"))

suma= (Numero1+Numero2)
resta= (Numero1-Numero2)
producto= (Numero1+Numero2)
Cociente= (Numero1/Numero2)

print("El resultado de la suma es", suma)
print("El resultado de la resta es",resta)
print("El resultado del producto es", producto)
print("El resultado del cociente es", Cociente)

Punto 6

In [ ]:
Ppetroleo = [76.24, 77.88, 78.31, 77.22, 76.71, 76.45, 77.17]
PlitroCoca = 1.09 #Precio en dolares.
PbarrilCoca = 0.00628981 * 1.09
for i in Ppetroleo:
  proporcion = i/PlitroCoca
  print("Precio petroleo: ", i, "Precio barril de cocaCola: ", PbarrilCoca, "Proporcion: ", proporcion)


Precio petroleo:  76.24 Precio barril de cocaCola:  0.0068558929000000005 Proporcion:  69.94495412844036
Precio petroleo:  77.88 Precio barril de cocaCola:  0.0068558929000000005 Proporcion:  71.44954128440367
Precio petroleo:  78.31 Precio barril de cocaCola:  0.0068558929000000005 Proporcion:  71.8440366972477
Precio petroleo:  77.22 Precio barril de cocaCola:  0.0068558929000000005 Proporcion:  70.8440366972477
Precio petroleo:  76.71 Precio barril de cocaCola:  0.0068558929000000005 Proporcion:  70.37614678899081
Precio petroleo:  76.45 Precio barril de cocaCola:  0.0068558929000000005 Proporcion:  70.13761467889908
Precio petroleo:  77.17 Precio barril de cocaCola:  0.0068558929000000005 Proporcion:  70.79816513761467


Observaciones: con un precio fluctuante, el barril de petroleo aumentó ligeramente su precio en los ultimos dias, pasando de valer de alrededor de 69.4 litros de coca a 70.8.